### ISSUE TITLE
fix tooltips they are blinking like crazy!

### ISSUE THREAD

**Issue by larp0** [aldrin-labs/solana-borrow-lending#51]
> implement keyboard navigation, add comprehensive keyboard tooltips everywhere
> overall UX should be pushing user to start using keyboard navigation by showing shortcuts how to do things on each screen
> kinda like command+k but for defi dapp
> 
>  

 _Originally posted by @larp0 in [#49](https://github.com/aldrin-labs/solana-borrow-lending/issues/49#issuecomment-3068210362)_

fix tooltips they are blinking like crazy!
also cmd+k doesnt work on linux, try something else! and show in a left bottom corner a floating button that will open a memo of hot-keys!
